dataeng/jobs/analytics/SnowflakePublicGrantsCleaner.groovy

```diff
@@ -26,7 +26,6 @@ class SnowflakePublicGrantsCleaner {
                 env('USER', allVars.get('USER'))
                 env('ACCOUNT', allVars.get('ACCOUNT'))
             }
-            logRotator common_log_rotator(allVars)
```

Comment:
While looking at this file I noticed that `logRotator` is already defined on line 17. This just removes the duplicate line since I was already making changes in this repo.